src/screens/CreateAttendanceSheet/CreateAttendanceScreen.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet, View,
+} from 'react-native';
+import { Button } from 'react-native-elements';
+import Firestore from '@react-native-firebase/firestore';
+import PropTypes from 'prop-types';
+import CalendarPicker from 'react-native-calendar-picker';
+
+const styles = StyleSheet.create({
+  hidden: {
+    display: 'none',
+  },
+});
+
+export default function CreateAttendanceScreen({ navigation }) {
+  const currentDate = new Date();
```

Comment:
check the role of the user before allowing the user to create an attendance sheet, return to the homescreen if the user is not a teacher